app/src/main/java/org/p2p/wallet/swap/ui/jupiter/settings/presenter/SwapContentSettingsMapper.kt

```diff
@@ -0,0 +1,242 @@
+package org.p2p.wallet.swap.ui.jupiter.settings.presenter
+
+import java.math.BigDecimal
+import org.p2p.core.common.DrawableContainer
+import org.p2p.core.common.TextContainer
+import org.p2p.core.utils.formatToken
+import org.p2p.uikit.components.finance_block.FinanceBlockCellModel
+import org.p2p.uikit.components.finance_block.FinanceBlockStyle
+import org.p2p.uikit.components.left_side.LeftSideCellModel
+import org.p2p.uikit.components.right_side.RightSideCellModel
+import org.p2p.uikit.model.AnyCellItem
+import org.p2p.uikit.utils.image.ImageViewCellModel
+import org.p2p.uikit.utils.skeleton.SkeletonCellModel
+import org.p2p.uikit.utils.text.TextViewCellModel
+import org.p2p.uikit.utils.toPx
+import org.p2p.wallet.R
+import org.p2p.wallet.swap.jupiter.domain.model.SwapTokenModel
+import org.p2p.wallet.swap.jupiter.repository.model.JupiterSwapRoute
+import org.p2p.wallet.swap.jupiter.repository.model.JupiterSwapToken
+import org.p2p.wallet.swap.jupiter.statemanager.SwapStateManager
+import org.p2p.wallet.utils.Base58String
+import org.p2p.wallet.utils.emptyString
+
+class SwapContentSettingsMapper(
+    private val commonMapper: SwapCommonSettingsMapper
+) {
+
+    fun mapForLoadingTransactionState(
+        slippage: Double,
+        routes: List<JupiterSwapRoute>,
+        activeRoute: Int,
+        jupiterTokens: List<JupiterSwapToken>,
+        tokenB: SwapTokenModel,
+    ): List<AnyCellItem> = mapList(slippage, routes, activeRoute, jupiterTokens, null, tokenB)
+
+    fun mapForSwapLoadedState(
+        slippage: Double,
+        routes: List<JupiterSwapRoute>,
+        activeRoute: Int,
+        jupiterTokens: List<JupiterSwapToken>,
+        tokenBAmount: BigDecimal?,
+        tokenB: SwapTokenModel,
+    ): List<AnyCellItem> = mapList(slippage, routes, activeRoute, jupiterTokens, tokenBAmount, tokenB)
+
+    private fun mapList(
+        slippage: Double,
+        routes: List<JupiterSwapRoute>,
+        activeRoute: Int,
+        jupiterTokens: List<JupiterSwapToken>,
+        tokenBAmount: BigDecimal?,
+        tokenB: SwapTokenModel,
+    ): List<AnyCellItem> = mutableListOf<AnyCellItem>().apply {
+        addRouteCell(routes, activeRoute, jupiterTokens)
+        this += commonMapper.getNetworkFeeCell()
+        addAccountFeeCell()
```

Comment:
i think it's better to have add in one way

so everything is
this +=
this +=
this +=

or everything is 

addLiquidityFeeCell
addEstimatedFeeCell
addMinimumReceivedCell